OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/RatingDialogFragment.java

```diff
@@ -184,4 +205,12 @@ public void persistRatingAndAppVersion(final float rating) {
         appPrefs.setAppRating(rating);
         appPrefs.setLastRatedVersion(BuildConfig.VERSION_NAME);
     }
+
+    public interface OnCancelListener {
+        /**
+         * This callback will be called when user will close the rating dialog without performing
+         * some action (i-e- open play store for rating and giving feedback).
```

Comment:
Some mistakes in the description are as follows:
- when **the** user will close
- performing **any** action
- `i.e.` instead of `i-e-`
- rating **or** giving feedback